tests/phpunit/test-wc-stripe-payment-gateway-legacy-methods.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * These tests make assertions against abstract class WC_Stripe_Payment_Gateway for non UPE legacy payment methods
+ * Testing with Giropay as payment method
+ *
+ * @package WooCommerce_Stripe/Tests/WC_Stripe_Payment_Gateway
+ */
+
+
+class WC_Stripe_Payment_Gateway_Legacy_Methods_Test extends WP_UnitTestCase {
```

Comment:
To keep the test names consistent with the class names which they are testing, I think it'd be a good idea to move the tests in this class to the existing `WC_Stripe_Payment_Gateway_Test` class.

Reply:
Moved these 2 tests inside the `WC_Stripe_Payment_Gateway_Test` class in [this commit](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1774/commits/9c2c1c2829b5d8123a3902e1a9a1218d461d8d78). But we need to initiate another gateway in the class now (for any of the legacy methods), as the existing gateway is for `Stripe` which overrides the `admin_options` function and can not be used to assert the desired divs.

Reply:
Great catch and thank you for including this change!